js/apps/system/_admin/aardvark/APP/statistics.js

```diff
@@ -406,152 +407,57 @@ router.use((req, res, next) => {
-                } else {
-                  merged.http[value][counter2] += x;
-                }
-              } catch (err) {}
-              counter2++;
-            });
-          });
-          _.each(arrays, function (value) {
-            counter2 = 0;
-            _.each(stat[value], function (x) {
-              try {
-                if (merged[value][counter2] === undefined) {
-                  merged[value][counter2] = 0;
-                } else {
-                  merged[value][counter2] += x;
-                }
-              } catch (err) {}
-              counter2++;
-            });
-          });
-        }
-        counter++;
-      }
-    });
-  };
-
-  var coordinators = global.ArangoClusterInfo.getCoordinators();
-  var coordinatorStats;
-  if (Array.isArray(coordinators)) {
-    coordinatorStats = coordinators.map(coordinator => {
-      const op = ArangoClusterComm.asyncRequest(
-        "GET",
-        "server:" + coordinator,
-        "_system",
-        '/_admin/aardvark/statistics/short',
-        "",
-        {},
-        { timeout: 10 }
-      );
-
-      const r = ArangoClusterComm.wait(op);
-
-      if (r.status === "RECEIVED") {
-        res.set("content-type", "application/json; charset=utf-8");
-        return JSON.parse(r.body);
-      } 
-      if (r.status === "TIMEOUT") {
-        throw new httperr.BadRequest("operation timed out");
-      } 
-      let body;
-      try {
-        body = JSON.parse(r.body);
-      } catch (e) {
-        // noop
-      }
-
-      return {};
-
-      /* don't throw here, as this will render the entire web UI cluster stats broken */
-      /*
-      throw Object.assign(
-        new httperr.BadRequest("error from Coordinator '" + coordinator + "', possibly Coordinator unknown or down"),
-        {extra: {body}}
-      );
-      */
-    });
-
-    if (coordinatorStats) {
-      mergeHistory(coordinatorStats);
+  var merged = { };
+
+  const coordinators = global.ArangoClusterInfo.getCoordinators();
+  try {
+    const stats15Query = `
+      FOR s IN _statistics15
+        FILTER s.time > @start
+        FILTER s.clusterId IN @clusterIds
+        SORT s.time
+        LET clientConnectionsCurrent = s.client.httpConnections
+        LET time = s.time
+        COLLECT clusterId = s.clusterId INTO server KEEP clientConnectionsCurrent, time
+        LET clientConnectionsCurrent = LAST(server).clientConnectionsCurrent
+        COLLECT AGGREGATE clientConnections15M = SUM(clientConnectionsCurrent)
+        RETURN {clientConnections15M: clientConnections15M || 0}`;
+
+    const statsSampleQuery = `
+      FOR s IN _statistics
+        FILTER s.time > @start
+        FILTER s.clusterId IN @clusterIds
+        RETURN {
+          time: s.time,
+          clusterId: s.clusterId,
+          avgRequestTime: s.client.avgRequestTime,
+          bytesSentPerSecond: s.client.bytesSentPerSecond,
+          bytesReceivedPerSecond: s.client.bytesReceivedPerSecond,
```

Comment:
IMHO this should also return `physicalMemory`, `residentSize` etc., as these are accessed by `MergeStatistics(...)`.